scripts/circle-ci/run-circle-tests.sh

```diff
@@ -62,10 +65,15 @@ else
     export CASSANDRA_HEAP_NEWSIZE=24m
 fi
 
+ETE_EXCLUDES=('-x :atlasdb-ete-tests:longTest')
+if [[ $INTERNAL_BUILD != true ]]; then
+    ETE_EXCLUDES+=('-x :atlasdb-ete-tests:timeLockTest')
```

Comment:
Are we not running this on external circle because it doesn't work? If so, I think we should add a comment. If we move to CircleCI 2.0, this might work for us.